app/frontend/src/components/ChatLayout/ChatLayout.tsx

```
@@ -1,29 +1,42 @@
             <div className={styles.chatRoot}>
                 <div className={styles.chatContainer}>
                     {showExamples ? (
                         <div className={styles.chatEmptyState} tabIndex={0}>
-                            <h2 className={styles.chatEmptyStateSubtitle}>{header}</h2>
+                            {header_as_markdown ? (
+                                <div className={styles.chatEmptyStateSubtitleMarkdown}>
+                                    <Markdown
+                                        className={styles.answerText}
+                                        remarkPlugins={[remarkGfm]}
+                                        rehypePlugins={[rehypeRaw]}
+                                        children={header}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)